.github/workflows/integration-test.yml

```
@@ -144,3 +144,52 @@ jobs:
           path: dist/*.tar.gz
           if-no-files-found: error
           retention-days: 90
+  docker-image-test:
+    runs-on: ubuntu-latest
+    # set a dependency sequence to occur after build job completion
+    needs: build
+    steps:
+      # checks out selected files for docker image build testing
+      - name: Checkout selected files
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            build/docker/Dockerfile
+            tests
+          sparse-checkout-cone-mode: false
+      - name: Download sdist artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: sdist
+      # gather the sdist tar.gz name (which varies)
+      - name: Get sdist tar.gz
+        id: get_sdist_tar_gz
+        run: |
+          echo "filename=$(ls ./*.tar.gz)" >> "$GITHUB_OUTPUT"
+      # unzips the sdist
+      - name: Extract sdist
+        run: |
+          tar -xzvf "${{ steps.get_sdist_tar_gz.outputs.filename }}"
```

```suggestion
          tar -xzvf "${{ env.sdist_filename}}" --directory pycytominer
```
Recommend using env var and

---

Thanks, way better!